pkg/api/norman/customization/gke/listers.go

```diff
@@ -0,0 +1,184 @@
+package gke
```

Comment:
Just scanning through these, do any of them have changes for gkev2 ? For instance where is shared VPC's ? Any differences between v1 and v2 will have to be handled separately, as v1 is not changing at all so it must work the same as it does today.

Reply:
Both `gkev1` and `gkev2` were integrated here

Reply:
I have to checked shared VPC function @cmurphy do you have idea how I get shared VPC list ?

Reply:
That's moving the goal posts on this PR a little, isn't it? I thought this PR was about using cloud credentials on the existing proxy methods.

There's no all-encompassing GET API for shared VPCs. You can get networks for a project and you can get the [shared VPC host project for a project](https://cloud.google.com/compute/docs/reference/rest/v1/projects/getXpnHost). What do you imagine this proxy method would return, how would the UI look? We could make recursive calls to build a tree of projects and networks...we should have that discussion as a new feature in a new PR, not as an addon to this one.

I don't see any other methods needed here for gke v2.

Reply:
I agree that implementation is done.